modulefiles/module_base.gaeac5.lua

```diff
@@ -15,6 +15,7 @@ load(pathJoin("cdo", (os.getenv("cdo_ver") or "None")))
 load(pathJoin("hdf5", (os.getenv("hdf5_ver") or "None")))
 load(pathJoin("netcdf-c", (os.getenv("netcdf_c_ver") or "None")))
 load(pathJoin("netcdf-fortran", (os.getenv("netcdf_fortran_ver") or "None")))
+load(pathJoin("esmf", (os.getenv("esmf_ver") or "None")))
```

Comment:
Does ESMF actually need to be loaded at runtime?  Usually, this library is statically linked.

Reply:
I think I tried it without, and it didn't work. Let me double-check that.

Reply:
OK, then I'm fine with keeping it.  Resolving conversation.